templates/github/.github/workflows/ci.yml.j2

```diff
@@ -66,12 +66,6 @@ jobs:
           - TEST: docs
           - TEST: s3
 
-    services:
-      postgres:
-        image: postgres:9.6
-      redis:
-        image: redis
```

Comment:
I guess these are "unused" because we build our own container?

Reply:
Yea, it runs in our single container.